starter-code/index.html

```diff
@@ -1,31 +1,55 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="./css/main.css">
   <title>Instagram</title>
 </head>
+
 <body>
-  Instagram
+    <div class="font"></div>
+  <div class="both-content">
+      <div class="left-content">
+        <img src="images/phones.png">
```

Comment:
Here, since you only have one element inside your div, you could have simply put your `<img />` tag with a class of "left content" or something, and it would have been fine :)